docs/source/torch.compiler_deepdive.rst

```diff
@@ -109,6 +112,17 @@ If you want to understand better what TorchDynamo is doing, you can run your cod
 
    TORCH_LOGS="+dynamo,guards,bytecode"
 
+If you are not familiar with Python bytecode, you can add a decompiler hook
+to decompile the bytecode into human-readable source code. One available
+tool is `depyf <https://github.com/youkaichao/depyf>`__. If you don't have
+``depyf`` already installed, run ``pip install depyf``. Then, add the
+following code to install decompilation hooks before you run any code.
+
+.. code-block:: python
+
+   import depyf
+   depyf.install()
```

Comment:
Just a nit - calling install everytime is a bit counter intuitive. Might I suggest `depyf.init()`?

Reply:
Is there also an overhead to installing? In which case having `depyf.disable()` would be good too.

Reply:
We have `depyf.uninstall()` function, too. You can refer to the repo https://github.com/thuml/depyf .

Reply:
It indeed adds some overheads to compilation. So it's mainly used for understanding and debugging.

Reply:
Cool. Would still suggest `depyf.enable()`, `depyf.disable()` as the API instead as `install()/uninstall()` is confusing and might be confused with `pip install`-ing something.

Reply:
I would vote for `register_hook()/remove_hook()`. But changing this API requires another PR to change the doc. I used to request approval from @jansel , but it seems he is not available recently.

Reply:
@youkaichao, hmm I see. I think the pytorch team is busy at pytorch conference https://pytorch.org/events. They should be more responsive by next week.

I think you can keep but deprecate the old API, and release the new API in a new release, and recommend new API changes in a PR in pytorch repo.

Reply:
This hasn't been released (in a stable branch) yet, so we are free to make changes.

Reply:
I'm thinking of refactoring the deep dive doc to something like https://github.com/thuml/depyf#example-usage . The dumped src is easy to understand and contains almost all the information of Dynamo. The dump src functionality is much more useful than just bytecode hooks. WDYT? @jansel @jon-chuang